spec/spec_helper.rb

```diff
@@ -91,6 +91,7 @@
       mocks.verify_partial_doubles = true
     end
     rspec_config.exclusion_filter = :perm
+    rspec_config.filter_run_excluding opi: true
```

Comment:
could this not be combined with the `perm` exclusion?

Reply:
We’re not that familiar with the CC codebase and ruby and we’re not sure we understand what the perm exclusion is representing in this case. To us it looks like the perm tests are testing something completely different and in this case it wouldn’t make much sense to combine them. What do you think?